packages/debug/src/browser/editor/debug-breakpoint-widget.tsx

```diff
@@ -92,23 +93,30 @@ export class DebugBreakpointWidget implements Disposable {
         this.toDispose.push(monaco.modes.CompletionProviderRegistry.register({ scheme: input.uri.scheme }, {
             provideCompletionItems: async (model, position, context, token) => {
                 const suggestions = [];
-                if ((this.context === 'condition' || this.context === 'logMessage')
-                    && input.uri.toString() === model.uri.toString()) {
+                if ((this.context === 'condition') || (this.context === 'logMessage' && DebugBreakpointWidget.isCurlyBracketOpen(inputEditor)) &&
```

Comment:
It looks like you've changed the bracketing here. Just to be explicit, I'd add parentheses back around the `||` conditions. I might even extract those conditions to named variables, since the condition ends up being pretty long and hard to follow.

Reply:
Agreed, I've made some changes based on your suggestions